### PR TITLE
Adding Bahasa Indonesia translation for certificates page

### PR DESCRIPTION
**Pull request by yofriadi**
addresses #15341

/language id
/assign @girikuncoro 
/assign @irvifa

### CODE REVIEW

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+**easyrsa** secara manual dapat menghasilkan sertifikat untuk kluster kamu.
+
+1. Unduh, buka paket, dan inisialisasi versi tambal easyrsa3.
+
+        curl -LO https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz
+        tar xzf easy-rsa.tar.gz
+        cd easy-rsa-master/easyrsa3
+        ./easyrsa init-pki
+1. Hasilkan CA. (`--batch` atur mode otomatis. `--req-cn` _default_ CN untuk digunakan.)
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: `--batch` **untuk** atur mode otomatis.`--req-cn` untuk **menggunakan** _default_ CN

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+        ./easyrsa --batch "--req-cn=${MASTER_IP}@`date +%s`" build-ca nopass
+1. Hasilkan sertifikat dan kunci _server_.
+    Argumen `--subject-alt-name` menetapkan IP yang mungkin dan nama DNS yang akan diakses
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: Argumen `--subject-alt-name` digunakan untuk mengatur alamat IP dan nama DNS yang dapat diakses oleh _server_ API.

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    oleh _server_ API. `MASTER_CLUSTER_IP` biasanya merupakan IP pertama dari layanan CIDR
+    yang ditentukan sebagai argumen` --service-cluster-ip-range` untuk _server_ API dan
+    komponen manajer pengontrol. Argumen `--days` digunakan untuk mengatur jumlah hari setelah
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: mengatur jumlah hari masa berlaku sertifikat.

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+    sertifikat berakhir.
+    Sampel di bawah ini juga mengasumsikan bahwa kamu menggunakan `cluster.local` sebagai standar
+    nama ranah DNS.
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: I prefer to use: nama _domain_ DNS _default_ than translate it to Bahasa Indonesia.

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+secara manual melalui `easyrsa`, `openssl` atau `cfssl`.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+### easyrsa
+
+**easyrsa** secara manual dapat menghasilkan sertifikat untuk kluster kamu.
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: **easyrsa** dapat digunakan untuk menghasilkan sertifikat kluster kamu secara manual.

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+        --tls-private-key-file=/yourdirectory/server.key
+
+### openssl
+
+**openssl** secara manual dapat menghasilkan sertifikat untuk kluster kamu.
+
+1.  Hasilkan ca.key dengan 2048bit:
+
+        openssl genrsa -out ca.key 2048
+1.  Dari ca.key menghasilkan ca.crt (gunakan -days untuk mengatur waktu efektif sertifikat):
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: Hasilkan ca.crt berdasarkan ca.key

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+1. Hasilkan server.key dengan 2048bit:
+
+        openssl genrsa -out server.key 2048
+1. Buat _file_ konfigurasi untuk menghasilkan _Certificate Signing Request_ (CSR).
+    Pastikan untuk mengganti nilai yang ditandai dengan kurung sudut (mis. `<MASTER_IP>`)
+    dengan nilai sebenarnya sebelum menyimpan ke _file_ (mis. `csr.conf`).
+    Perhatikan bahwa nilai `MASTER_CLUSTER_IP` adalah layanan IP kluster untuk
+    _server_ API seperti yang dijelaskan dalam subbagian sebelumnya.
+    Sampel di bawah ini juga mengasumsikan bahwa kamu menggunakan `cluster.local`
+    sebagai standar nama ranah DNS.
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: See above. Prefer to not translate to Bahasa

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+        IP.1 = <MASTER_IP>
+        IP.2 = <MASTER_CLUSTER_IP>
+
+        [ v3_ext ]
+        authorityKeyIdentifier=keyid,issuer:always
+        basicConstraints=CA:FALSE
+        keyUsage=keyEncipherment,dataEncipherment
+        extendedKeyUsage=serverAuth,clientAuth
+        subjectAltName=@alt_names
+1. Hasilkan permintaan penandatanganan sertifikat berdasarkan file konfigurasi:
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: Changes file to _file_

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+        }
+1.  Hasilkan kunci CA (`ca-key.pem`) dan sertifikat (`ca.pem`):
+
+        ../cfssl gencert -initca ca-csr.json | ../cfssljson -bare ca
+1.  Buat _file_ konfigurasi JSON untuk menghasilkan kunci dan sertifikat untuk API
+    _server_, misalnya, `server-csr.json`. Pastikan untuk mengganti nilai dalam kurung sudut
+    dengan nilai sebenarnya yang ingin kamu gunakan. `MASTER_CLUSTER_IP` adalah layanan
+    kluster IP untuk _server_ API seperti yang dijelaskan dalam subbagian sebelumnya.
+    Sampel di bawah ini juga mengasumsikan bahwa kamu menggunakan `cluster.local` sebagai
+    standar nama ranah DNS.
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: See above. Prefer to not translate to Bahasa

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 10]
```diff
@@ -0,0 +1,250 @@
...
+        }
+1.  Buat kunci dan sertifikat untuk server API, yang mana awalnya di
+    simpan masing-masing ke dalam _file_ `server-key.pem` dan `server.pem`:
+
+        ../cfssl gencert -ca=ca.pem -ca-key=ca-key.pem \
+        --config=ca-config.json -profile=kubernetes \
+        server-csr.json | ../cfssljson -bare server
+
+
+## Distributing Self-Signed CA Certificate
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: Distribusi Sertifikat _Self-Signed_ CA

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 11]
```diff
@@ -0,0 +1,250 @@
...
+```
+
+```
+Updating certificates in /etc/ssl/certs...
+1 added, 0 removed; done.
+Running hooks in /etc/ca-certificates/update.d....
+done.
+```
+
+## Certificates API
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: Sertifikat API

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 12]
```diff
@@ -0,0 +1,250 @@
...
+
+Kamu dapat menggunakan API `Certificate.k8s.io` untuk menyediakan
+sertifikat x509 yang digunakan untuk autentikasi seperti yang didokumentasikan
+[disini](/docs/tasks/tls/managing-tls-in-a-cluster).
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: di sini

File: content/id/docs/concepts/cluster-administration/certificates.md [hunk 13]
```diff
@@ -0,0 +1,250 @@
...
+        curl -LO https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz
+        tar xzf easy-rsa.tar.gz
+        cd easy-rsa-master/easyrsa3
+        ./easyrsa init-pki
+1. Hasilkan CA. (`--batch` atur mode otomatis. `--req-cn` _default_ CN untuk digunakan.)
+
+        ./easyrsa --batch "--req-cn=${MASTER_IP}@`date +%s`" build-ca nopass
+1. Hasilkan sertifikat dan kunci _server_.
+    Argumen `--subject-alt-name` menetapkan IP yang mungkin dan nama DNS yang akan diakses
+    oleh _server_ API. `MASTER_CLUSTER_IP` biasanya merupakan IP pertama dari layanan CIDR
```
Comment (danninov) [inline content/id/docs/concepts/cluster-administration/certificates.md]: ... biasanya merupakan IP pertama dari CIDR _service cluster_ yang diset dengan argumen ...

**Comment by yofriadi** [comment]
Hi @danninov I'm curious if there is any standardization process on how to decide to pick one over another in PR or we just trusts the reviewer.
Also one thing since I have involved in this project, may I get write permissions for the glossary, there is some word that don't appear in glossary but shows up in here and get directed to be translated on some word, I think I should contribute to that too.

Thank you

**Comment by danninov** [comment]
> Hi @danninov I'm curious if there is any standardization process on how to decide to pick one over another in PR or we just trusts the reviewer.
> Also one thing since I have involved in this project, may I get write permissions for the glossary, there is some word that don't appear in glossary but shows up in here and get directed to be translated on some word, I think I should contribute to that too.
> 
> Thank you

Do you mean whether to accept the suggestion from reviewer or reject it? As far as I know, there is no standardization yet. For more discussion please go to our Slack Channel

**Comment by irvifa** [comment]
@yofriadi Hi, it’s on kubernetes’ slack channel sig-docs-id once the review process is over the reviewer can ping me or Giri to approve the PR so that it’ll be merged into master. Cheers..

**Comment by yofriadi** [comment]
Hi @danninov I have updated the translation based on your suggestion. Thank you.

**Comment by girikuncoro** [comment]
Great work @yofriadi , thanks for your contribution! 🎉  We are very happy if you could help us translate other pages as well 😄 

/lgtm
/approve

**Pull request merged**